src/sql/initDb.sql

```diff
@@ -25,18 +25,25 @@ END$$;
 CREATE TABLE IF NOT EXISTS df.news_feed
 (
     account varchar(48) NOT NULL,
-    activity_id bigint NOT NULL
+    activity_id bigint NOT NULL UNIQUE
```

Comment:
We don't need `UNIQUE` here, multiple `account`s can have notifications about the same `activity_id`.
```suggestion
    activity_id bigint NOT NULL
```